packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -2924,6 +2924,13 @@ const schema: SchemaType<"Users"> = {
     optional: true
   },
 
+  activeRateLimits: {
+    type: GraphQLJSON,
+    canRead: ['guests'],
```

Comment:
this is definitely a `canRead: ['sunshineRegiment', 'admins']` type of situation

Reply:
also I think it ought to just be a `resolverOnlyField`?

Reply:
Oh, I see, you need access to `getFeatures` in the resolver logic, which is server-only code